weekly planner/index.html

```diff
@@ -0,0 +1,123 @@
+                </li>
+            </ul>
+        </div>
+        <div class="day">
+            <header>Wednesday</header>
+            <ul>
+                <li>
+                    <div class="event-title">9:00 AM - Standup Meeting</div>
+                    <div class="event-description">Daily standup meeting with the team.</div>
+                </li>
+                <li>
+                    <div class="event-title">12:00 PM - Lunch Break</div>
+                    <div class="event-description">Lunch break.</div>
+                </li>
+                <li>
+                    <div class="event-title">4:00 PM - Team Meeting</div>
+                    <div class="event-description">Weekly team meeting to discuss progress.</div>
+                </li>
+            </ul>
+        </div>
+        <div class="day">
+            <header>Thursday</header>
+            <ul>
+                <li>
+                    <div class="event-title">9:00 AM - Project Planning</div>
+                    <div class="event-description">Plan the next phases of the project.</div>
+                </li>
+                <li>
+                    <div class="event-title">11:00 AM - Development</div>
+                    <div class="event-description">Focus on development tasks.</div>
+                </li>
+                <li>
+                    <div class="event-title">3:00 PM - Sprint Review</div>
+                    <div class="event-description">Review the sprint progress and achievements.</div>
+                </li>
+            </ul>
+        </div>
+        <div class="day">
+            <header>Friday</header>
+            <ul>
+                <li>
+                    <div class="event-title">10:00 AM - Code Review</div>
+                    <div class="event-description">Review the latest code submissions.</div>
+                </li>
+                <li>
+                    <div class="event-title">1:00 PM - One-on-One</div>
+                    <div class="event-description">One-on-one meeting with the manager.</div>
+                </li>
+                <li>
+                    <div class="event-title">4:00 PM - Happy Hour</div>
+                    <div class="event-description">Relax and unwind with colleagues.</div>
+                </li>
+            </ul>
+        </div>
+        <div class="day">
+            <header>Saturday</header>
+            <ul>
+                <li>
+                    <div class="event-title">Free Time</div>
+                    <div class="event-description">Enjoy your day off!</div>
+                </li>
+            </ul>
+        </div>
+        <div class="day">
+            <header>Sunday</header>
+            <ul>
+                <li>
+                    <div class="event-title">Family Day</div>
+                    <div class="event-description">Spend quality time with family.</div>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+</body>
```

Comment:
good starting point for our html, but since it is hard coded we will need to revise later. Also consider using semantic html instead of the divs with different classes. We can use section, header and, make the tasks buttons so they are clickable.